File: app/src/main/java/com/wealoha/social/beans/PostLikeNotify2.java
package com.wealoha.social.beans;

import java.util.List;

/**
 * 
 * 
 * @author javamonk
 * @createTime 2015年2月25日 上午10:57:11
 */
public class PostLikeNotify2 extends AbsNotify2 {

	/**
	 * 
	 */
	private static final long serialVersionUID = -1517780633471613601L;
	private final Post post;
	private final List<User2> user2s;
	private int count;
	public PostLikeNotify2(Notify2Type type, boolean unread, String notifyid, long updateTimeMillis, Post post, List<User2> user2s, int count) {
		super(type, unread, notifyid, updateTimeMillis);
		this.post = post;
		this.user2s = user2s;
		this.count = count;
	}

	public int getCount() {
		return count;
	}

	public void setCount(int count) {
		this.count = count;
	}

	public Post getPost() {
		return post;
	}

	public List<User2> getUser2s() {
		return user2s;
	}
}

File: app/src/main/java/com/wealoha/social/beans/PostLikeNotify2.kt
package com.wealoha.social.beans

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 *
 *
 * @author javamonk
 * @createTime 2015年2月25日 上午10:57:11
 */
@Parcelize
data class PostLikeNotify2(
    override var type: Notify2Type,
    val unread: Boolean = false,
    var notifyid: String?,
    override var updateTimeMillis: Long,
    private val post: Post,
    private val user2s: List<User>,
    private var count: Int
) : AbsNotify2(type, unread, notifyid, updateTimeMillis), Parcelable {
    fun getCount(): Int {
        return count
    }

    fun setCount(count: Int) {
        this.count = count
    }

    fun getPost(): Post {
        return post
    }

    fun getUsers(): List<User> {
        return user2s
    }

    companion object {
        /**
         *
         */
        private const val serialVersionUID = -1517780633471613601L
    }

}